data/human/human missions.txt

```diff
@@ -3601,3 +3601,58 @@ mission "Mafia Extortion"
 			label payup
 			`	The man takes your credits, responds with a brief "Thank you," and walks away.`
 				accept
+	
+mission "Thrall Emigration"
+	name "Thrall Emigration"
+	description "Transport a family from Thrall to <destination>, where they hope to find better employment. They cannot pay you."
+	minor
+	source
+		planet "Thrall"
+	destination "Delve"
+	to offer
+		random < 50
+	passengers 2
+	on offer
+		conversation
+			`By the edge of the spaceport at night, you see a thin sweaty man turn away from a pilot, disheartened. You make eye contact, and he begins hurrying over to you.`
+			choice
+				`	(Wait for him.)`
+				`	(Walk away quickly.)`
+					decline
+			`	"Excuse me! Are you a pilot?" Without waiting for a response, he continues "I need to go to the Syndicate. I work all day picking cotton with these hands, but my family's still starving. We can't afford transport..."`
+			`	You can barely understand him through his accent. The man's teeth look worse than his clothes, and his breath makes you step back. His eyes are human.`
```

Comment:
What was your intent with including `His eyes are human`? As-is, it comes off more like a strange confirmation that this person isn't an alien. I think this could maybe use some rewording if this wasn't your intended meaning.

Reply:
```suggestion
			`	You can barely understand him through his accent. The man's teeth look worse than his clothes, and his breath makes you step back. His eyes implore you.`
```
The original intent was to establish commonality after the previous description, but I was thinking in terms of empathy rather than species. The wording is kind of awkward as well: would something like this work better? 
Thanks for all the comments. I committed your other suggestions.

Reply:
That works great!